recipe/meta.yaml

```diff
@@ -5,43 +5,55 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/nco/nco/archive/{{ version }}.tar.gz
-  sha256: 5ba930f00a0e9775f85748d145acecfe142f917c180de538b2f8994788446cf8
+  # url: https://github.com/nco/nco/archive/{{ version }}.tar.gz
+  # sha256: 5ba930f00a0e9775f85748d145acecfe142f917c180de538b2f8994788446cf8
+  # Just testing before we have a new release.
+  git_url: https://github.com/nco/nco.git
+  git_branch: master
 
 build:
   number: 4
-  skip: True  # [win]
+  skip: True  # [win32 or (win and py<=35)]
+  features:
+    - vc14  # [win and (py35 or py36)]
 
 requirements:
   build:
-    - gsl >=2.2,<2.3
+    - python  # [win]
+    - cmake  # [win]
     - antlr >=2.7.7,<3
+    - bison  # [not win]
+    - texinfo  # [not win]
+    - flex  # [not win]
+    - m4  # [not win]
     - curl >=7.44.0,<8
-    - zlib 1.2.11
+    - expat 2.2.*
+    - gsl >=2.2,<2.3
     - hdf5 1.10.1
+    - krb5 1.14.*  # [not win]
     - libnetcdf 4.4.*
     - udunits2
-    - expat 2.2.*
-    - krb5 1.14.*
-    - texinfo
-    - bison
-    - flex
-    - m4
+    - zlib 1.2.11
+    - vc 14  # [win and (py35 or py36)]
   run:
+    - curl >=7.44.0,<8
+    - expat 2.2.*
     - gsl >=2.2,<2.3
     - hdf5 1.10.1
+    - krb5 1.14.*  # [not win]
     - libnetcdf 4.4.*
     - udunits2
-    - expat 2.2.*
-    - libgcc
-    - esmf
-    - openblas 0.2.20|0.2.20*
+    - libgcc  # [not win]
+    - openblas 0.2.20|0.2.20*  # [not win]
+    - esmf  # [not win]
+    - vc 14  # [win and (py35 or py36)]
 
 test:
   source_files:
-    - data/in.nc
+    - data/in.cdl
   commands:
     - ncks --help
+    - ncks -M "http://tds.marine.rutgers.edu/thredds/dodsC/roms/espresso/2013_da/his/ESPRESSO_Real-Time_v2_History_Best"
     - ncap2 --help
```

Comment:
@czender a simple `ncap2` call is working on Windows. See  https://ci.appveyor.com/project/ocefpaf/nco-feedstock/build/1.0.55#L2008